src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -317,7 +320,13 @@ const drawDataLines = (scale, config, lineGroupSVG) =>
         .classed(styles.line, true)
         .append("path")
         .attr("d", (value) => createLine(scale, value))
-        .attr("style", (value) => `stroke: ${getColorForTarget(value)};`)
+        .attr(
+            "style",
+            (value) =>
+                `stroke: ${getColorForTarget(value)}; stroke-dasharray: ${
+                    value.strokeDasharray
```

Comment:
`value.style.strokeDashArray`?

Reply:
Updated after changing the way I am storing the style.